get_facts_from_openstack_workaround.yml

```diff
@@ -0,0 +1,20 @@
+- name: Collect facts from Openstack instances - workaround
+  hosts: all
+  vars:
+  tasks:
+  - include_vars:
+      # file: "{{ lookup('file', '/var/tmp/openstack_creds') }}"
+      file: '/var/tmp/openstack_creds'
```

Comment:
Do you create this file when you execute the playbook?

Reply:
@sbulage 
Please add your view on this question.

Reply:
@sbulage @ganeshhubale I really want to understand if each of the playbook you are adding in this PR is going to be executed? 
If so, have you tested it ? Can you please share the results? Thanks.

Reply:
hello @kedark3 
I have automated test case which uses these playbooks.
Please have a look: https://github.com/ManageIQ/integration_tests/pull/9222